### PR TITLE
Replaced our causes with workshops

### PR DESCRIPTION
**Pull request by nabinNath**
Screenshot (111)
Screenshot (112)

### CODE REVIEW

**Comment by Dishebh** [comment]
Please share a preview link for the pr. Also, I see some grammatical errs/ bugs in sentence framing, please fix that as well.

File: index.html [hunk 1]
```diff
@@ -259,51 +256,97 @@ <h5 class="card-title">Fourth created forth fill
...
 <div class="card-body">
 <a href="#">
-<h5 class="card-title">Fourth created forth fill
-created subdue be </h5>
+<h5 class="card-title">Poster Making</h5>
 </a>
 <ul>
-<li><img src="images/passion_1.svg" alt=""> Goal: $2500</li>
-<li><img src="images/passion_2.svg" alt=""> Raised: $1533</li>
+<li>As poster making is a necessary and in NIMBUS it is used as a way to attract people for various events we organised a workshop on poster making and photoshop was taughted</li>
```
Comment (Dishebh) [inline index.html:265]: For ref: 
```suggestion
<li>As poster making is necessary and in NIMBUS it is used as a way to attract people for various events we organised a workshop on poster making and photoshop was taught</li>

```

**Comment by Dishebh** [comment]
Also mention the issue number it fixes, if any.

**Comment by nabinNath** [comment]
mistakes corrected

**Pull request merged**